dev/RadioButtons/APITests/RadioButtonsTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using MUXControlsTestApp.Utilities;
+
+using Windows.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls;
+using Common;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+    [TestClass]
+    public class RadioButtonsTests : ApiTestBase
+    {
+        [TestMethod]
+        public void ValidateSelectionChangedEvent()
+        {
+            bool item1InitiallySelected = false;
```

Comment:
I added the term "Initially" to indicate that no item was selected yet. That said, the way the name reads now doesn't really reflect that. Something like "item1SelectedAsFirstSelection" would more properly describe that but that might be a bit too long...then again, I think this would be fine.

In the end, "item1Selected" should be fine as well.

Reply:
Does no longer exist since switching to a switch case logic.